technical/get-started/quickstart.mdx

```diff
@@ -82,5 +82,12 @@ Download [example workflows](https://github.com/livepeer/comfystream/tree/main/w
   <Info>
     The ComfyStream server logs are in the ComfyUI log window. Open the bottom panel using the **Toggle Bottom Panel** button at the top-right of the interface.
   </Info>
+</Accordion>
 
+<Accordion title="Issue connecting to comfystream when using runpod">
+   If you encounter issue when starting comystream in runpod 
+   ``` bash
+   Error proxying offer: Cannot connect to host localhost:8889 ssl:default [Connect call failed ('127.0.0.1', 8889)]
+   ```
+   Please check the [server settings](/technical/get-started/install#accessing-comfystream) to ensure those allign with the [Stream URL](/technical/get-started/quickstart#comfystream-ui-preview-window) set in the UI.
```

Comment:
```suggestion
   Please check the [ComfyStream Server Settings](/technical/get-started/install#accessing-comfystream) to ensure ComfyStream is running on host `0.0.0.0` when using RunPod. The default **Stream URL** in ComfyStream UI (`http://127.0.0.1:8889`) is correct when using the [livepeer-comfystream](RunPod Template](http://localhost:3001/technical/get-started/install#run-with-runpod).
```
I've also posted some troubleshooting steps for this issue here https://github.com/yondonfu/comfystream/issues/199,  We could also include a section about ComfyStream UI in more detail.